Client/src/components/layout/Search/Searchbar.jsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState, useEffect } from 'react';
+import Input from '../Form/Input/Input';
+import Button from '../Buttons/Button';
+import axios from 'axios';
+import styles from './Searchbar.modules.css';
+import getSearchedCity from '../../../api/getSearchedCity';
+import SearchResults from './SearchResults';
+
+function Searchbar() {
+  const [search, setSearch] = useState('');
+  const [elapsedTime, setElapsedTime] = useState(Boolean);
+  const [token, setToken] = useState();
+  const [searchResult, setSearchResult] = useState();
+
+  let timer,
```

Comment:
optionaly, you could benefit from using `lodash.debounce` I guess

Reply:
yes but I didn't want to install additional things just for one function